data/json/itemgroups/activities_hobbies.json

```
@@ -656,15 +633,7 @@
       [ "gloomhaven", 5 ],
       [ "mansions_of_madness", 5 ],
       [ "world_nations", 5 ],
-      [ "terraforming_mars", 5 ],
       [ "terraforming_mercury", 5 ],
```

```suggestion
      [ "terraforming_mars", 5 ],
```
Wouldn't keeping Mars be better? I don't know whether this references something, but terraforming Mars is more realistic than Mercury since it's more like the Earth and a more comfortable distance from the Sun.

---

I see now that you are not removing it, you are just changing this to item variants. Feel free to close, if you see no benefit for the base item to be Mars.

---

It's also a real game.